python/cudf/cudf/_libxx/aggregation.pyx

```diff
@@ -5,12 +5,21 @@ import numba
 import numpy as np
 from libcpp.string cimport string
 from libcpp.memory cimport unique_ptr
+from libcpp.vector cimport vector
 from cudf.utils import cudautils
 
-from cudf._libxx.lib cimport *
-from cudf._libxx.lib import *
+from cudf._libxx.types import np_to_cudf_types, cudf_to_np_types
+from cudf._libxx.move cimport move
+
+from cudf._libxx.aggregation cimport get_aggregation
+from cudf._libxx.cpp.types cimport (
+    type_id,
+    size_type,
+    data_type,
+    interpolation
+)
+cimport cudf._libxx.cpp.aggregation as cudf_aggregation
```

Comment:
This feels like an awkward naming convention.

Reply:
```suggestion
cimport cudf._libxx.cpp.aggregation as cpp_aggregation
```

Reply:
Agreeg. But also everywhere else that we do

`cimport cudf._libxx.cpp.types as cudf_types`

should we do instead:

`cimport cudf._libxx.cpp.types as cpp_types`

I think that can be confusing

Reply:
Naming is hard 😞, I think we should just pick a prefix and stick with it for all externed functions / modules. I'm fine with `cpp_*` or `libcudf_*` but `cudf_*` is super ambiguous.

Reply:
`cpp_` is even more so. `libcudf_` makes the most sense to me